qutip/cy/brtools.pxd

```diff
@@ -1,37 +1,3 @@
-#!python
```

Comment:
This `#!` line in a .pxd file seems pretty strange, so I guess removing it is a good idea!

Reply:
Technically outside scope, but I left this change in anyway because this is definitely a mistake - the file's not even marked as executable, so it could never have had an effect anyway.